apps/music/style/music.css

```diff
@@ -193,6 +193,11 @@ body.tiles-mode #views {
   transform: translateX(0%);
 }
 
+body.tiles-mode #views-tiles,
+body.search-from-tiles-mode #views-tiles {
```

Comment:
`#views-tiles-search` lives inside `#views-tiles`, so that won't work. I think you are right that the actual tiles should be hidden, so I'll fix that with a new rule.